src/components/NewClusterModal.js

```diff
@@ -111,7 +111,7 @@ export const NewClusterModal = withModalDrawer({ width: 675 })(class NewClusterM
     this.state = {
       profile: matchingProfile ? matchingProfile.name : 'custom',
       jupyterUserScriptUri: '',
-      isCustomEnv: false, customEnvImage: '', viewMode: undefined,
+      customEnvImage: '', viewMode: 'JupyterEnv', prevViewMode: 'JupyterEnv',
```

Comment:
`prevViewMode` is more important when the `delete warn view` is added to the drawer since you will be able to delete from any of the environment views (Jupyter, Custom, and Other) having the `prevViewMode` ensures that clicking the `back arrow` or `back button` will return the user to the correct view.

Reply:
Instead of this, why not build an array of where you are and have been, breadcrumb style?

Reply:
I hadn't considered that, but it sounds like a more straightforward approach, I will look into that. Thanks!